tests/product/resources/uninstall_twice.txt

```diff
@@ -13,36 +13,36 @@ Warning: [slave3] Presto is not installed.
 
 Fatal error: [slave2] sudo() received nonzero return code 1 while executing!
 
-Requested: rpm -e presto
-Executed: sudo -S -p 'sudo password:'  /bin/bash -l -c "rpm -e presto"
+Requested: rpm -e presto-server-rpm
+Executed: sudo -S -p 'sudo password:'  /bin/bash -l -c "rpm -e presto-server-rpm"
 
 Aborting.
-[slave2] out: error: package presto is not installed
+[slave2] out: error: package presto-server-rpm is not installed
 [slave2] out:
 
 Fatal error: [master] sudo() received nonzero return code 1 while executing!
 
-Requested: rpm -e presto
-Executed: sudo -S -p 'sudo password:'  /bin/bash -l -c "rpm -e presto"
+Requested: rpm -e presto-server-rpm
```

Comment:
This needs to be a regex that is substituted in, since it can be either presto-server-rpm or presto. See base_product_case.replace_keywords() for something you can do about it